pipeline-runner/R/gem2s-1-download_user_files.R

```diff
@@ -32,25 +31,65 @@ download_user_files <- function(input, pipeline_config, prev_out = list()) {
       fname <- basename(gem_key)
 
       # Preparing directories
-      local_fpath <- file.path(input_dir, sample, fname)
-      fs::dir_create(dirname(local_fpath))
+      local_fpath <- file.path(input_dir, sample_id, fname)
+
+      download_and_store(originals_bucket, gem_key, local_fpath, aws_config)
+    }
+  }
+}
 
-      # Download the file and store the output in a variable
-      c(body, ...rest) %<-% s3$get_object(
-        Bucket = pipeline_config$originals_bucket,
-        Key = gem_key
-      )
+file_names_v1 <- list(
+  barcodes10x = "barcodes.tsv.gz",
+  features10x = "features.tsv.gz",
+  matrix10x = "matrix.mtx.gz"
+)
 
-      # Write output to file
-      writeBin(body, con = local_fpath)
+file_types_by_technology <- list(
```

Comment:
if `file_types_by_technology` and `file_types_by_technology` are constants I'd make it clearer either by moving them somewhere else like the top of the file, another constants file or upper case (I don't know what is the convention in pipeline code). If the list `file_types_by_technology` needs to match another list in the API I would add a comment too.